docs/millennium-os/manual/chapters/20_installation.md

```diff
@@ -86,45 +86,45 @@ If you experience any connection issues during the wizard, you can usually reloa
     If you realise this during the configuration wizard process, don't worry - you can resume where you left off once fixing the issue.
 
 !!! note
-    If you experience any connection issues during the wizard, you can usually reload your browser tab to trigger the next dialog.
+    If you experience connection issues during the wizard, you can reload your browser tab to trigger the next dialog.
```

Comment:
```suggestion
    If you experience connection issues during the wizard, you can typically reload your browser tab to trigger the next dialog.
```

This block looks to be the same as the one above, which was changed to add `typically`. I'm not sure we need both, but if we do, I like the addition of `typically`.

If we don't need both, then I think go with the annotation version (`!!! note`) and in the location that makes the most sense in the file.